src/resources/packages/classy/store/constants.ts

```diff
@@ -0,0 +1 @@
+export const STORE_NAME = 'tribe/tickets/classy'; 
```

Comment:
Following the convention, the names should be:
* Common -> `tec/classy`
* TEC -> `tec/classy/events`
* ECP -> `tec/classy/events-pro`
* ET -> `tec/classy/tickets`
* ET+ -> `tec/classy/tickets-plus`

Reply:
I agree, the store was left in a basic state after AI created it, and I still ned to clean it up.